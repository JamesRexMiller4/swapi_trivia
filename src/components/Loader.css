.loader-section {
  width: 100%;
  height: 100vh;
  background-color: black;
}

.loader-h2 {
  width: inherit;
  font-family: 'Titillium Web', sans-serif;
  font-size: 30px;
  color: #0befff;
}

.loader-div-wave {
  display: inline-block;
	margin-top:50vh;
	text-align:center;
	width:100px;
	height:100px;
	margin-left: auto;
  margin-right: auto;
}
.loader-div-wave .dot {
  display:inline-block;
  width:6px;
  height:6px;
  border-radius:50%;
  margin-right:3px;
  background:#0befff;
  animation: wave 1.3s linear infinite;
}

.loader-div-wave .dot:nth-child(2) {
  animation-delay: -1.1s;
}

.loader-div-wave .dot:nth-child(3) {
  animation-delay: -0.9s;
}

@keyframes wave {
  0%, 60%, 100% {
    transform: initial;
  }
  30% {
    transform: translateY(-15px);
  }
}