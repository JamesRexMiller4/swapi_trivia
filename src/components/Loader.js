import React from 'react';
import './Loader.scss';


const Loader = () =>  {
  return (
    <section className='loader-section'>
      <h2 className='loader-h2'>A long time ago in a galaxy far far away
      <div className="loader-div-wave">
        <span className="dot"></span>
        <span className="dot"></span>
        <span className="dot"></span>
      </div></h2>
    </section>
  )
}

export default Loader;
