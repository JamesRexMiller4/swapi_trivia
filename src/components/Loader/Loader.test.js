import React from 'react';
import ReactDOM from 'react-dom';
import Loader from './Loader';
import {shallow} from 'enzyme';

describe('Loader', () => {

  it.skip('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Loader />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
});