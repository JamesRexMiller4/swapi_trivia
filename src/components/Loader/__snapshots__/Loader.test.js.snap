// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot with all the data passed through 1`] = `
<section
  className="loader-section"
>
  <h2
    className="loader-h2"
  >
    A long time ago in a galaxy far far away
    <div
      className="loader-div-wave"
    >
      <span
        className="dot"
      />
      <span
        className="dot"
      />
      <span
        className="dot"
      />
    </div>
  </h2>
</section>
`;
